src/app/refs_page.rs

```diff
@@ -104,21 +99,13 @@ impl<'repo> Navigable<'repo> for RefsPage<'repo> {
 
     fn pagedown(&mut self, page_size: u16) {
         let h: usize = page_size.into();
-        let selected_index = self.selected_index + h;
-        self.selected_index = if selected_index > self.len() {
-            self.len() - 1
-        } else {
-            selected_index
-        }
+        let index = self.selected_index.saturating_add(h);
```

Comment:
I think this should be the same as the previous `if`, except saturating_add/sub prevents underflows/overflows. 🤔